tests/end_to_end/test_suricata_integration/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,72 @@
+- name: Prepare environment
+  hosts: wazuh-manager
+  become: true
+  vars:
+    suricata_conf_path: /etc/suricata/suricata.yaml
+  tasks:
+
+    - name: Configure Wazuh to read Suricata logs file
+      blockinfile:
+        path: /var/ossec/etc/ossec.conf
+        marker: <!-- {mark} READ SURICATA LOGS CONFIG -->
+        insertbefore: ^</ossec_config>
+        block: |
+          <localfile>
+          <log_format>syslog</log_format>
+          <location>/var/log/suricata/eve.json</location>
+          </localfile>
+
+    - name: Restart Wazuh to apply the change
+      systemd:
+        state: restarted
+        name: wazuh-manager
+
+    - name: Check if Suricata is installed
+      shell: dpkg -l suricata | grep suricata
+      register: check_suricata
+      ignore_errors: true
+
+    - name: Add and configure the repo to install Suricata
+      shell: |
+        add-apt-repository ppa:oisf/suricata-5.0 -y
+        apt-get update -y
+      when: '"no packages found matching suricata" in check_suricata.stderr'
+
+    - name: Install Suricata
+      package:
+        name: suricata=5.0.9-0ubuntu4
+        state: present
+      when: '"no packages found matching suricata" in check_suricata.stderr'
+
+    - name: Download and extract Emerging rules
+      shell: |
+        cd /tmp/
+        curl -LO https://rules.emergingthreats.net/open/suricata-5.0.8/emerging.rules.tar.gz
+        tar -xvzf emerging.rules.tar.gz && mv rules/*.rules /etc/suricata/rules/
+        chmod 640 /etc/suricata/rules/*.rules
+      when: '"no packages found matching suricata" in check_suricata.stderr'
+
+    - name: Change the default interface
+      replace:
+        path: "{{ suricata_conf_path }}"
+        regexp: 'af-packet:\n  - interface: eth0'
+        replace: 'af-packet:\n  - interface: {{ ansible_default_ipv4.interface }}'
+
+    - name: Configure external network in Suricata
+      replace:
+        path: "{{ suricata_conf_path }}"
+        regexp: 'EXTERNAL_NET: "!\$HOME_NET"'
+        replace: 'EXTERNAL_NET: "any"'
+
+    - name: Configure rules path in Suricata
+      replace:
+        path: "{{ suricata_conf_path }}"
+        regexp: 'default-rule-path: \S.*\n\nrule-files:\n.*- suricata.rules'
+        replace: 'default-rule-path: /etc/suricata/rules\n\nrule-files:\n  - "*.rules"'
+
+    - name: Start Suricata
+      systemd:
+        daemon_reload: true
+        enabled: true
+        state: started
```

Comment:
We need to restart Suricata

Reply:
Done in aff0d49, the task has not changed, but the way Suricata is installed has.